tlsr9/crypto/mbedtls/internal/compatibility/ecp_alt.c

```diff
@@ -83,8 +83,16 @@
 #include <string.h>
 
 #if defined(MBEDTLS_ECP_ALT)
+#if CONFIG_SOC_RISCV_TELINK_B91 || CONFIG_SOC_RISCV_TELINK_B92
 #include <pke.h>
-#if (CONFIG_SOC_RISCV_TELINK_B92 || CONFIG_SOC_RISCV_TELINK_B95)
+#elif CONFIG_SOC_RISCV_TELINK_B95
+#include <lib/include/pke/pke.h>
+
+#include <ext_driver/driver_lib/ext_lib.h>
+#include <ext_driver/ext_misc.h>
+#endif
```

Comment:
why we can't use same path for all SoCs?

Reply:
for the driver file changes , and we don't want to to adjust the drive file .

Reply:
ok